server_config_parser_test.cc

```diff
@@ -74,11 +74,22 @@ class HandlerParseTest : public ::testing::Test {
 };
 
 TEST_F(HandlerParseTest, NoHandlersConfig) {
-    ASSERT_FALSE(getNumberOfHandlersParsed("no /handlers;"));
+    ASSERT_FALSE(GetNumberOfHandlersParsed("no /handlers;"));
     EXPECT_EQ(num_parsed_handlers, 0);
 }
 
 TEST_F(HandlerParseTest, SomeHandlersConfig) {
-    ASSERT_TRUE(getNumberOfHandlersParsed("echo /echo;\nstatic /static ./;"));
+    ASSERT_TRUE(GetNumberOfHandlersParsed("echo /echo;\nstatic /static ./;"));
     EXPECT_EQ(num_parsed_handlers, 2);
 }
+
+TEST_F(HandlerParseTest, ImproperEchoHandlerConfig) {
+    ASSERT_FALSE(GetNumberOfHandlersParsed("echo ;\n echo echo;"));
+    EXPECT_EQ(num_parsed_handlers, 0);
+}
+
+TEST_F(HandlerParseTest, ImproperStaticHandlerConfig) {
+    ASSERT_FALSE(GetNumberOfHandlersParsed("static ;\nstatic static /"));
```

Comment:
The string is missing a semicolon at the end so it is "improper" in the wrong way